### DISCUSSION TITLE
Using pyright to separate code into valid lines, def, class ...

### DISCUSSION THREAD

**Discussion by giangtranducts** [microsoft/pyright#5638] [Q&A] [answered]
Can I use pyright to separate code into valid lines, def, class?
example, 3 lines bellow should separate as one valid line:
```
texts = ["""Crawling the lazy dog.
The quick brown fox jumps over the lazy dog (mua mua mua mua)""",
"From my_app import views"] 
```
Same with def and class, do we have any feature to do that?

**Comment by erictraut** [comment, marked as answer]
Pyright is a static type checker. So no, it doesn't have any features that reformat your code.
